include/dlaf/lapack_tile.tpp

```diff
@@ -13,6 +13,7 @@
 template <class T, Device device>
 void hegst(int itype, blas::Uplo uplo, const Tile<T, device>& a, const Tile<T, device>& b) {
   DLAF_ASSERT(a.size().rows() == a.size().cols(), "Error: HEGST: matrix A is not square.");
+  DLAF_ASSERT(itype > 0 && itype < 4, "Only itype = 1, 2, 3 allowed", itype);
```

Comment:
Just remarking that `DLAF_ASSERT(itype >= 0 && itype <=3, itype);`, IMHO, it would have been more concise and clear.

Reply:
My only concern is that in the description of hegst parameters of the error message, only `int` appears, not also `itype`:
```
[ERROR] /home/lara/Documents/DLAF/DLA-Future-trans2stdeigen/include/dlaf/lapack_tile.tpp:16 : void dlaf::tile::hegst(int, blas::Uplo, const dlaf::Tile<T, device>&, const dlaf::Tile<T, device>&) [with T = float; dlaf::Device device = (dlaf::Device)0]
itype < 0 && itype <= 3
5
```
Is it clear enough in this case?